### PR TITLE
Add link feature to the userguide

### PR DESCRIPTION
**Pull request by Howlong11**

### CODE REVIEW

**Review by Murugan-Maniish — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -312,6 +317,14 @@ Format: `lookup INDEX`
 Examples:
 * `lookup 1` returns all persons associated with the first article in the list of articles.
 
+
+### [3.2.9. Opening webpage for an article](#32-managing-articles)
+
+By clicking the `Link` button of your article that is highlighted in yellow box in the picture below, you can open up the webpage for your article that is added when you added the article.
```
Comment (Ko-Khan) [inline docs/UserGuide.md:323]: Consider adding an * here

File: docs/UserGuide.md [hunk 2]
```diff
@@ -290,15 +294,16 @@ Format: `filter -a S/Status TAG/Tag ST/Start date EN/End date`
...
 Remove filters so all articles are displayed.
 
 Format: `rmfilter -a`
 
 * No additional parameters.
 * The -a is necessary, additional letters will cause command to fail.
 
-### [3.2.6. Lookup for associated persons : `lookup`](#32-managing-articles)
+
+### [3.2.8. Lookup for associated persons : `lookup`](#32-managing-articles)
```
Comment (Ko-Khan) [inline docs/UserGuide.md:306]: Thanks for helping with toc

File: docs/UserGuide.md [hunk 3]
```diff
@@ -278,7 +282,7 @@ Format: `find -a KEYWORD [MORE_KEYWORDS]`
 
 Examples:
 * `find -a Monkey King` returns two articles: `Monkey breaking window of NTU student’s hall after being aggravated` and `King Charles’ health crisis: the future of Britain becomes uncertain`
-### [3.2.6 Filtering Articles]()
+### [3.2.6. Filtering Articles](#32-managing-articles)
```
Comment (Ko-Khan) [inline docs/UserGuide.md:285]: Thanks for fixing this

**Review by Ko-Khan — COMMENTED** [review]
I noticed that you did not tell the user how to add links?

File: docs/UserGuide.md (hunk 1 above)
Comment (Howlong11) [reply docs/UserGuide.md]: Done!

**Pull request merged**